lib/zebra.h

```diff
@@ -466,7 +466,8 @@ typedef enum {
   AFI_IP  = 1,
   AFI_IP6 = 2,
   AFI_ETHER = 3,                /* RFC 1700 has "6" for 802.* */
-  AFI_MAX = 4
+  AFI_L2VPN = 4,
```

Comment:
This breaks the rfapi codes usage AFI_ETHER (die to ethernet AF being mapped to new AFI).  It's fine to define the new type, but all previous usage of AFI_ETHER needs to be changed to AFI_L2VPN or the 2 AFI values need to be the same.